styleguide_example/common/services.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Dict, Any, Tuple
+
+from styleguide_example.common.types import DjangoModelType
+
+
+def model_update(
+    *,
+    instance: DjangoModelType,
+    fields: List[str],
+    data: Dict[str, Any]
+) -> Tuple[DjangoModelType, bool]:
+    """
+    Generic update service meant to be reused in local update services
+
+    For example:
+
+    def user_update(*, user: User, **data) -> User:
+        fields = ['first_name', 'last_name']
+        user = model_update(instance=user, fields=fields, data=data)
+
+        // Do other actions with the user here
+
+        return user
+
+    Return value: Tuple with the following elements:
+        1. The instance we updated
+        2. A boolean value representing whether we performed an update or not.
+    """
+    has_updated = False
+
+    if instance is None:
```

Comment:
Do we need this, since `instance` is not `Optional` :thinking:

Reply:
You're right, now that we have correct typings we do not need this check 👍🏼